ssl/t1_lib.c

```diff
@@ -997,6 +997,24 @@ static const SIGALG_LOOKUP *tls1_get_legacy_sigalg(const SSL *s, int idx)
                     }
                 }
             }
+            /*
+             * Here is another fallback: when broken implementations did not sent
+             * proper signature_algorithm extension, we try to use this function.
```

Comment:
This first sentence seems redundant here and is seems to apply to the whole function.

Reply:
Removed. Thanks!